tests/ui/linkage-attr/linkage-detect-local-generated-name-collision.rs

```diff
@@ -1,32 +0,0 @@
-<<<PULL-UPSTREAM>>> file deleted upstream; move the Ferrocene annotations if any, and delete this file
-//@ build-fail
-// FIXME(#83838) codegen-units=1 triggers llvm asserts
-//@ compile-flags: -Ccodegen-units=16
-#![feature(linkage)]
-
-mod dep1 {
-    extern "C" {
-        #[linkage = "external"]
-        #[no_mangle]
-        pub static collision: *const i32; //~ ERROR symbol `collision` is already defined
-    }
-}
-
-#[no_mangle]
-pub static _rust_extern_with_linkage_collision: i32 = 0;
-
-mod dep2 {
-    #[no_mangle]
-    pub static collision: usize = 0;
-}
-
-fn main() {
-    unsafe {
-        println!("{:p}", &dep1::collision);
-    }
-}
-
-// ferrocene-annotations: fls_mvd7nz8k3wcy
-// Attribute no_mangle
-//
-// ferrocene-annotations: um_rustc_C_codegen_units
```

Comment:
Do we have a new home we can give these annotations?

Reply:
oh, there should be a bunch... should I do that here or in a follow-up

Reply:
Here is good